pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -70,12 +75,19 @@ python3.pkgs.buildPythonApplication rec {
       url = "https://github.com/mesonbuild/meson/commit/a38ad3039d0680f3ac34a6dc487776c79c48acf3.patch";
       hash = "sha256-9YaXwc+F3Pw4BjuOXqva4MD6DAxX1k5WLbn0xzwuEmw=";
     })
-  ];
+  ]
+    # Nixpkgs cctools does not have bitcode support.
+    ++ lib.optional stdenv.isDarwin ./disable-bitcode.patch;
```

Comment:
Ideally we would make the patch in a way that it works on all platforms and is not forgotten when someone on linux updates meson.